src/styles/partials/components/_collapse.scss

```diff
@@ -34,4 +34,8 @@
   .dg_accordion_buttontext-holder {
     margin-left: $margin-large;
   }
+
+  @media screen and (max-width: $medium-breakpoint) {
```

Comment:
@martypowell @tmccoy529 Does the spacing issue occur at tablet size as well/ do we want to cover that potential occurrence? If so, I would set the breakpoint to `$large-breakpoint` If not, this looks fine as-is.

Reply:
so actually it looks like we want it there at all times not on a break point. So this has been updated to reflect that instead.